### DISCUSSION TITLE
About sponsor 关于赞助

### DISCUSSION THREAD

**Discussion by tonylu110** [tonylu110/uyou-todo-electron#34] [General]
If you have sponsored `uyou ToDo`, you can bring your name and sponsorship screenshots in subsequent discussions, and this list will be added to `uyou ToDo`.

Or use [QQ Channel](https://pd.qq.com/s/6uaqgzd25?shareSource=5) tell me.

如果你赞助过`uyou ToDo`，你可以在后续的讨论中带上你的名字和赞助截图，后续这个列表将加入到`uyou ToDo`中显示

或者使用 [QQ 频道](https://pd.qq.com/s/6uaqgzd25?shareSource=5) 告诉我

**Comment by FEI352** [comment]
名字：Hsieh Chi Jenn
截图 
![Pasted image 20240525020912](https://github.com/tonylu110/uyou-todo-electron/assets/104716040/718eb716-d394-4191-98db-62511d271656)

**Reply by tonylu110** [reply]
Added

**Comment by 18lenda** [comment]
姓名：18lenda
截图：
![1](https://github.com/user-attachments/assets/feabdda7-9086-4aa9-9e46-fc1d6a8bda99)

**Reply by tonylu110** [reply]
Added

**Comment by lingyuajnzi** [comment]
姓名：lingyuanzi
截图：
![c3c945bcfee33f68c73a779fa96e3759](https://github.com/user-attachments/assets/6a0732f8-dd37-4948-b537-3e40e725ee78)

**Reply by yunzhongyue23** [reply]
![4e043c2ebd83b5c42ee2f59a361d1ae6](https://github.com/user-attachments/assets/afbd07a0-031f-484a-938d-ebbf4606493f)

**Comment by acbiong** [comment]
姓名：BIong
![6a5b11e6de23e9ca83b1ab605e886ba5](https://github.com/user-attachments/assets/4467e33a-d5f2-43e2-90da-b4351e87977e)
